tests/manage/test_expand_cluster_by_adding_backend_vol_rbd.py

```diff
@@ -0,0 +1,378 @@
+
+    self.pvcs = pvc.create_multiple_pvc(number_of_pvcs_to_create, **pvc_data)
+
+    # Create the app pods on these PVCs and Run fio based IOs from them
+    self.pods = []
+
+    # change the following loop contents to use new way of creating pods. get new pod.py
+    number_of_pods_to_create = 2  # TBD: change this to parameterized format later. Accept it from cmdline
+    for i in range(number_of_pods_to_create):
+        self.pods.append(helpers.create_pod(
+                                                constants.CEPHBLOCKPOOL,
+                                                self.pvcs[i].name,
+                                                constants.STATUS_RUNNING,
+                                                True
+                                        ))
+
+
+def check_ocs_pods_status():
+    """
+    This function checks the status of all the pods in the openshift-storage namespace. All pods should be either
+    in 'Running' or 'Completed' state.
+
+    Returns :
+        pod_health: boolean value. True if all the pods are in either 'Running' or 'Completed/Succeeded' state.
+                    False, otherwise
+
+    """
+    list_of_pods = pod.get_all_pods(defaults.ROOK_CLUSTER_NAMESPACE)
+    pod_health = True
+    for p in list_of_pods:
+        pod_dict = p.get()
+        if pod_dict['status']['phase'] not in constants.STATUS_RUNNING:
+            if pod_dict['status']['phase'] not in constants.STATUS_SUCCEEDED:
+                pod_health = False
+                logging.error(f"pod {p.name} in Unhealthy")
+    return pod_health
+
+
+def get_pod_restarts_count():
+    """
+    Gets the dictionary of pod and its restart count for all the ocs pods
+
+    Returns:
+        restart_dict: Dictionary of pod name and its corresponding restart count
+
+    """
+    list_of_pods = pod.get_all_pods(defaults.ROOK_CLUSTER_NAMESPACE)
+    restart_dict = {}
+    for p in list_of_pods:
+        pod_dict = p.get()
+        # we don't want to compare osd-prepare pod as it gets created freshly when an osd need to be added.
+        if "rook-ceph-osd-prepare" not in p.name:
+            restart_dict[p.name] = pod_dict['status']['containerStatuses'][0]['restartCount']
+
+    return restart_dict
+
+
+def add_osd_aws(
+    number_of_osds,
+    original_osd_count,
+    size=100
+):
+    """
+    Create volumes on workers
+
+    Args:
+        number_of_osds (int): number of osds to be added
+        original_osd_count (int): number of osds that were present before osd addition
+        size (int): Size in GB (default: 100)
+    """
+    osds_added = 0
+    cluster_path = config.ENV_DATA['cluster_path']
+
+    with open(os.path.join(cluster_path, "terraform.tfvars")) as f:
+        tfvars = json.load(f)
+
+    cluster_id = tfvars['cluster_id']
+    worker_pattern = f'{cluster_id}-worker*'
+    logger.info(f'Worker pattern: {worker_pattern}')
+
+    # aws = AWS(default.AWS_REGION)
+    aws = AWS(config.ENV_DATA['region'])
+
+    """
```

Comment:
It is ok to have description like that in the middle of the code but looks strange.
you can just use comment --> '#'
Of you can move this to the function description